nautobot/extras/plugins/__init__.py

```diff
@@ -623,3 +633,43 @@ def register_custom_validators(class_list):
             raise TypeError(f"PluginCustomValidator class {custom_validator} does not define a valid model!")
 
         registry["plugin_custom_validators"][custom_validator.model].append(custom_validator)
+
+
+def register_override_views(override_views, plugin):
+    resolver = get_resolver()
+
+    override_warning = (
+        "Plugin {} tried to override view {}, but only namespaced views "
+        "are supported (e.g. `dcim:device`, `plugins:myplugin:myview`)."
+    )
+
+    # This functionality will only work properly if core views are overridden before plugin views
+    # This sorts the dict by occurrences of ":" in the keys.
+    # Overridden plugin views will have two colons in the key so will be sorted to get processed last here:
+    override_views = dict(sorted(override_views.items(), key=lambda i: i[0].count(":")))
```

Comment:
Would a better fix be to avoid redefining the `resolver` variable inside the `for` loop (line 662) and instead use a local variable?